.python-version

```diff
@@ -0,0 +1 @@
+plone52
```

Comment:
I don't know what this `.python-version` file does. Is it created by virtualenv? It should probably be ignored then, as other people may have a different Python version, trying both 2.7 and 3.7 for example.
Also, `plone52` does not look like a Python version. :-)

Reply:
it's a pyenv file. Obviously we don't need it. Removed